puzzles/templates/modals/edit_status.html

```diff
@@ -1,41 +1,43 @@
-<div class="modal fade" id="editstatus" tabindex="-1" role="dialog" aria-labelledby="edittitle" aria-hidden="true">
-    <div class="modal-dialog" role="document">
-        <div class="modal-content">
-            <div class="modal-header">
-                <h5 class="modal-title" id="edittitle">Edit Status of <span id='puzzletitle'></span></h5>
-                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                    <span aria-hidden="true">&times;</span>
-                </button>
-            </div>
-            <form method="post" action="/puzzles/update_status/">
-                {% csrf_token %}
-                <div class="form-group">
-                    <select name="status" class="form-control form-control-sm">
-                        <option value="SOLVING" id='SOLVING'>SOLVING</option>
-                        <option value="STUCK" id='STUCK'>STUCK</option>
-                        <option value="EXTRACTION" id='EXTRACTION'>EXTRACTION</option>
-                    </select>
-                </div>
-                <div class="modal-footer">
-                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
-                    <button type="submit" class="btn btn-primary">Submit</button>
-                </div>
-            </form>
-        </div>
-    </div>
-</div>
-
-<script type="text/javascript">
-function initEditStatus() {
-  $(".editstatus").click(function() {
-    var pk = $(this).data("pk");
-    var name = $(this).data("name");
-    var status = $(this).data('status');
-    $('#editstatus form').attr('action', `/puzzles/update_status/${pk}/`);
-    $('#puzzletitle').text(name);
-    $('#' + status).attr('selected', 'selected');
-  });
-}
-initEditStatus();
-$('#puzzles').on('draw.dt', initEditStatus);
-</script>
+<div class="modal fade" id="editstatus" tabindex="-1" role="dialog" aria-labelledby="edittitle" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="edittitle">Edit Status of <span id='puzzletitle'></span></h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <form method="post" action="/puzzles/update_status/">
+                {% csrf_token %}
+                <div class="modal-body">
+                    <div class="form-group">
+                        <select name="status" class="form-control form-control-sm">
+                            <option value="SOLVING" id='SOLVING'>SOLVING</option>
```

Comment:
pre-existing, but why are these id's set to these all caps values? Are they needed?

Reply:
I think I added them to make it easier to get the DOM element using jQuery via `$('#<STATUS>')`. But looks like you can get by value using `$('#editstatus option[value="<STATUS>"])`. Removed the ids.